docs/1.3.5/docs/en/kubernetes-deployment.md

```diff
@@ -8,6 +8,16 @@
 
 ## Installing the Chart
 
+> NOTE: You need to change your `Chart.yaml` locally to make it work. Due to a change in the Bitnami repo,
+> https://charts.bitnami.com/bitnami was truncated only containing entries for the latest 6 months (from January 2022 on).
+> This URL: https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami contains the full 'index.yaml'.
+>
+> See detail here: https://github.com/bitnami/charts/issues/10833.
+>
+> Change you `Chart.yaml` in path `apache-dolphinscheduler-incubating-1.3.5-src-release/docker/kubernetes/dolphinscheduler` after you download from GitHub
```

Comment:
`after you download from GitHub` 

Is the download link of the official documentation not apache?